### PR TITLE
feat: Add filter by type in the curation page

### LINKED ISSUE #1724: Add type filters to Curation page
The current filters look like this:

We need to add a TYPE filter, and it should work like this:
- The TYPE filter should be `ALL`, `Standard`, `Third Party`
- If no TYPE filter is `ALL` or `Third Party`, both of the current selectors should be disabled
- If the TYPE is `Standard`, re-enable the other filters

If the TYPE changes, reset the other filters too

Depends on https://github.com/decentraland/builder-server/pull/382 for creating TP CollectionCurations
Might be related to #1723

This needs the server to support the filter as well. Create the task if necessary.

### PR DESCRIPTION
**Pull request by kevinszuchet**
Closes #1724 

image

### CODE REVIEW

File: src/components/CurationPage/CurationPage.css [hunk 1]
```diff
@@ -8,6 +8,12 @@
 
 .CurationPage .filters .text-filter-row {
   width: 100%;
+  box-sizing: border-box;
+  height: 44px;
+  background: #242129;
```
Comment (juanmahidalgo) [inline src/components/CurationPage/CurationPage.css:13]: We have a CSS variable that you can use for this color!

image

File: src/components/CurationPage/CurationPage.tsx [hunk 2]
```diff
@@ -142,6 +180,7 @@ export default class CurationPage extends React.PureComponent<Props, State> {
           { value: CurationFilterOptions.APPROVED, text: t('curation_page.filter.approved') },
           { value: CurationFilterOptions.REJECTED, text: t('curation_page.filter.rejected') }
         ]}
+        disabled={filterByType !== CollectionFilterOptions.STANDARD}
```
Comment (juanmahidalgo) [inline src/components/CurationPage/CurationPage.tsx:183]: Can we remove the _disabling_ logic? I think we support all filters for all types of collections now!

Comment (kevinszuchet) [reply src/components/CurationPage/CurationPage.tsx:183]: Yes, we can. We should also remove the `reset` of the filters after changing the type, should we?

**Review by juanmahidalgo — APPROVED** [review]
Looks great 🙌

**Review by mrbaguvix — APPROVED** [review]

**Review by juanmahidalgo — APPROVED** [review]
LGTM

**Pull request merged**